tests/test-definition.yaml

```diff
@@ -1,5 +1,7 @@
 ---
 dimensions: []
 tests:
-  - name: kerberos
```

Comment:
Marker to revert the change before merging

Reply:
8d28a9d6d6255eba2579116fa92abd5cbc8c633a